languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -342,6 +342,43 @@ title="Easy editing stylesheet" ?>
 
  <category id='GRAMMAR' name="Gramática Geral" type="grammar">
 
+<rule id="CONFUSION_OF_MAS_MAIS" name="confusion of mais and mas" default="temp_off">
+        <antipattern>
+        <token>mais</token>
+        <token postag='AQ.+' postag_regexp='yes'/>
+        </antipattern>
+        <antipattern>
+        <token>nunca</token>
+        <token>mais</token>
+        </antipattern>
+        <antipattern>
+        <token>quanto|quem</token>
+        <token>mais</token>
+        </antipattern>
+        <antipattern>
+        <token>quanto|quem</token>
+        <token>mais</token>
+        <token skip="-1">mais</token>
```

Comment:
`skip=-1` at the end of an `antipattern` does not make sense as it is irrelevant what comes after anyway